docs/source/conf.py

```diff
@@ -45,9 +45,14 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.ifconfig',
     'sphinx_exec_code',
-    'sphinx_tabs.tabs'
+    'sphinx_tabs.tabs',
+    "matplotlib.sphinxext.plot_directive",
 ]
 
+# Supress hyperlinks in the display of matplotlib.sphinxext.plot_directive plots
```

Comment:
suppress (sp)

Reply:
Oops. I'll fix it next time I touch that file.